packages/babel-traverse/src/path/index.js

```diff
@@ -170,8 +170,9 @@ Object.assign(
 
 for (const type of (t.TYPES: Array<string>)) {
   const typeKey = `is${type}`;
+  const fn = t[typeKey];
   NodePath.prototype[typeKey] = function(opts) {
-    return t[typeKey](this.node, opts);
+    return fn(this.node, opts);
   };
 
   NodePath.prototype[`assert${type}`] = function(opts) {
```

Comment:
In this function (next line) you coud use  `!fn(this.node, opts)` instead of `!this[typeKey](opts)` to skip one function call.

Reply:
Updated. Probably won't have much impact, but it's slightly cleaner.